### ISSUE TITLE
`first`: incorrect report in `.vue` file with 2 script tags

### ISSUE THREAD

**Issue by 9romise** [9romise/eslint-plugin-import-lite#8]
``` vue
<script lang="ts">
export interface Props {
  
}
</script>

<script lang="ts" setup>
import { UiButton } from '#components'
</script>

<template>
</template>
```
